extensions/wikia/Email/Controller/DiscussionController.class.php

```diff
@@ -48,20 +52,22 @@ public function body() {
             'details' => $this->getDetails(),
             'buttonText' => $this->getButtonText(),
             'buttonLink' => $this->postUrl,
-            'contentFooterMessages' => $this->getFooterMessages(),
+            'contentFooterMessages' => $this->getContentFooterMessages(),
             'hasContentFooterMessages' => true,
         ] );
     }
 
     protected abstract function getSummary();
 
-    protected abstract function getDetails();
+    protected function getDetails() {
+        return $this->postContent;
+    }
 
     private function getButtonText() {
         return $this->getMessage( 'emailext-discussion-button-text' )->text();
     }
 
-    protected function getFooterMessages() {
+    protected function getContentFooterMessages() {
```

Comment:
There are 2 similar named methods used to populate footer messages in emails. The first, `getFooterMessages`, is used to populate the messages at the far bottom of the email. The second, `getContentFooterMessages`, is used to populate the last messages in the main body of the email. I accidentally had those switched, so I renamed this to use the right method name.